bsi0007-tilt-sensor/include/TiltSensor.h

```diff
@@ -0,0 +1,17 @@
+#ifndef BSI0007_TILT_SENSOR_H
+#define BSI0007_TILT_SENSOR_H
+
+#include <AnalogPinReader.h>
+
+namespace wlp {
+    namespace BSI0007 {
+        class TiltSensor: public AnalogPinReader {
+        public:
+            explicit TiltSensor(Board::AnalogPin analog_pin)
+            : AnalogPinReader(analog_pin) {};
+            double read_value();
```

Comment:
you're implicitly overriding `read_value` from `AnalogPinReader` which isn't declared `virtual`. Can you update `read_value` in that header to be `virtual` and then stick `override` after this one?

Reply:
Oh right, `AnalogPinReader` isn't supposed to have that public method. I won't be able to test it if it doesn't, though, so I guess addinng the explciit override will work.